client/state/happychat/middleware.js

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
I thought I was going crazy, every time I pushed a change removing a newline this comment wouldn't collapse! Turns out GitHub was just moving your comment to the nearest empty line 😄